test/testframe.c

```diff
@@ -274,8 +276,35 @@ TestParseCmdLine(int argc, char *argv[])
             TestUsage();
             exit(EXIT_SUCCESS);
         }
-        else if ((HDstrcmp(*argv, "-cleanoff") == 0) || (HDstrcmp(*argv, "-c") == 0))
+        else if ((HDstrcmp(*argv, "-cleanoff") == 0) || (HDstrcmp(*argv, "-c") == 0)) {
```

Comment:
These changes to testframe are taken from Jordan's changes to the test framework in the upstream.